filament/backend/include/private/backend/Program.h

```diff
@@ -46,7 +47,12 @@ class Program {
         bool strict = false;        // if true, this sampler must always have a bound texture
     };
 
-    using SamplerGroupInfo = std::array<utils::FixedCapacityVector<Sampler>, BINDING_COUNT>;
+    struct SamplerGroup {
+        ShaderStageFlags stageFlags = ALL_SHADER_STAGE_FLAGS;
+        utils::FixedCapacityVector<Sampler> samplers;
```

Comment:
Swap those two lines, this is at least have a chance to reduce compile padding.

Reply:
Ok it's done.